CREATING_ISSUES.md

```diff
@@ -0,0 +1,135 @@
+# Creating GitHub issues
```

Comment:
How about merging with CONTRIBUTING.md? See https://contributing.md/how-to-build-contributing-md/ -- it's encouraged to have bug reporting section in one file with other contribution guidelines. Examples: https://github.com/rails/rails/blob/main/CONTRIBUTING.md https://github.com/atom/atom/blob/master/CONTRIBUTING.md#suggesting-enhancements